src/app/cli/interpreter.cpp

```diff
@@ -1443,9 +1444,19 @@ Interpreter::Value Interpreter::ProcessBr(const Expression &aExpr)
             }
         }
 
+        if (mContext.mCommandKeys.size() == 2 && mContext.mCommandKeys[0] == "--netif")
```

Comment:
Can we support multiple options e.g. "br scan --timeout 400 --netif wlan0"?

Reply:
Modified this part to support both options.